src/akkudoktoreosserver/flask_server.py

```diff
@@ -165,9 +164,9 @@ def flask_gesamtlast_simple():
         # leistung_wp = wp.simulate_24h(temperature_forecast)  # Simulate heat pump load for 24 hours
         # gesamtlast.hinzufuegen("Heatpump", leistung_wp)  # Add heat pump load to total load calculation
 
-        last = gesamtlast.gesamtlast_berechnen()  # Calculate total load
-        print(last)  # Output total load
-        return jsonify(last.tolist())  # Return total load as JSON
+        last = gesamtlast.calculate_total_load()  # Calculate total load
+        print(last)
```

Comment:
Using `logging.debug` instead of `print`?

Reply:
sure but my intention was not to clean up the flask_server for now. Lets make that a new PR